src/azul/indexer/document.py

```diff
@@ -1433,4 +1479,44 @@ def delete(self):
         return super().delete or not self.contents
 
 
+@document_class
+class Replica(Document[ReplicaCoordinates[E]]):
+    # The type of replica. Typically, all replicas of the same type have similar
```

Comment:
```suggestion
    # The type of replica. Typically, all replicas of the same type have similar
```

I had intentionally added colons in my REVIEW commit. It's Sphinx syntax.